src/main/java/seedu/address/logic/commands/FindCustomerCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
```

Comment:
Should we rename FindCommandParser to FindCustomerCommandParser? In order to be consistent